util/fusermount.c

```diff
@@ -25,6 +25,12 @@
 #include <sys/wait.h>
 #include <sys/stat.h>
 #include <sys/mount.h>
+
+/* uclibc doesn't define MS_LAZYTIME */
+#ifndef MS_LAZYTIME
+#define MS_LAZYTIME	(1<<25)
```

Comment:
Where does this value come from?

Reply:
Hi @Nikratio,

it comes from Linux header mount.h:
https://elixir.bootlin.com/linux/v6.2.8/source/include/uapi/linux/mount.h#L39
and it never changed value throug various versions since it's been added with Linux 4.0:
https://elixir.bootlin.com/linux/v4.0.9/source/include/uapi/linux/fs.h#L93

Unforunately uclibc doesn't add it to <sys/mount.h>. The other chance to fix the bug is to include Linux's:
[include/uapi/linux/mount.h](https://elixir.bootlin.com/linux/v5.0.21/source/include/uapi/linux/mount.h#L36)
but before Linux 5.0 that define is contained in:
https://elixir.bootlin.com/linux/v4.20.17/source/include/uapi/linux/fs.h#L133
So I don't know if that's a good idea. Or if we do that I can check against Linux version and include the 2 different headers.

Reply:
There's nothing wrong with hardcoding the value, I just want you to add the source to the (C) comment in the file :-)